### PR TITLE
Docs rewrite

### PR DESCRIPTION
**Pull request by danielballan**
This PR will restructure ophyd's documentation as [we just did for databroker](https://github.com/bluesky/databroker/pull/642).

It is _very_ early stages here, hardly worth reviewing yet, but I'm opening the PR so the work can proceed publicly.

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W
+SPHINXOPTS    = # -W
```
Comment (danielballan) [inline docs/Makefile:5]: Don't panic. This is temporary.

**Comment by klauer** [comment]
Looking better! 👍

File: docs/source/tutorials/epics/single-PV.rst [hunk 2]
```diff
@@ -6,33 +6,98 @@ In this tutorial we will read, write, and monitor an EPICS PV in ophyd.
...
 
-For this tutorial you will need an EPICS IOC serving a PV that you can connect
-to.  We suggest starting a simple test IOC using `caproto`_, as follows. If
-you already have another IOC handy, you may use that instead.
+Before you begin, install ``ophyd``, ``pyepics``, ``bluesky``, and ``caproto``,
+following the :doc:`install`.
 
-Install caproto.
+We'll start this simulated hardware that implements a `random walk`_. It has
+just two PVs. One is a tunable parameter, ``random_walk:dt``, the time between
```
Comment (prjemian) [inline docs/source/tutorials/epics/single-PV.rst:13]: Observation: So, technically, it's not a "single-PV" tutorial?

Comment (danielballan) [reply docs/source/tutorials/epics/single-PV.rst:13]: Hm. I meant it in the sense that we'll be using `EpicsSignal` and `EpicsSignalRO` to connect to bare PVs, as opposed to grouping them into ophyd Devices.

Comment (prjemian) [reply docs/source/tutorials/epics/single-PV.rst:13]: Right.  This follows a basic learning trajectory, use `EpicsSignal` (et al.) for PV connection, then progress to `Device` in next lesson.

Comment (danielballan) [reply docs/source/tutorials/epics/single-PV.rst:13]: I guess to me more precise I should title this “Single EPICS PVs” instead of “A Single EPICS PV”.

**Comment by danielballan** [comment]
The docs build without warnings now. The old content has been reorganized into the tutorial / how-to / explanations / reference scheme. Some new content has been added.

I intend to ask someone to summarily merge this on Monday so that we can refine and add from there in more reviewable PRs.

File: docs/source/tutorials/epics/single-PV.rst (hunk 2 above)
Comment (danielballan) [reply docs/source/tutorials/epics/single-PV.rst]: Done!

File: docs/Makefile (hunk 1 above)
Comment (danielballan) [reply docs/Makefile]: Reverted.

**Review by MikeHart85 — APPROVED** [review]
Looks great to me.

**Pull request merged**